doc/dynamic_discovery_configuration_guide.md

```diff
@@ -10,7 +10,8 @@ Doc. ID: UG-DDCG
  Date       | Version | Description                                                     | Author
  ---------- | ------- | --------------------------------------------------------------- | --------------------
  03.12.2021 | 1.0     | Initial version                                                 |
- 
+ 22.01.2022 | 1.1     | Add sections [2.3](#23-tls-configuration-and-certificates), [3.3](#33-registering-smp-in-sml) and [3.6.1](#361-tls-configuration-and-certificates). Minor updates, add more examples. | Petteri Kivimäki
```

Comment:
I would consider keeping the version comments more general and not include links to headings here. The reason is, that document structures tend to change over time and the links become broken in which case you need to consider if you want to "rewrite history" or live with the incorrect links. This is already an issue with the X-Road documentation in practice.

Reply:
That's a good point. Updated the description.